### DISCUSSION TITLE
Include slight delay in colour mode transitions

### DISCUSSION THREAD

**Discussion by orimdominic** [adityatelange/hugo-PaperMod#403] [Ideas]
The switch from light mode to dark mode and vice versa feature is awesome.
Is there a chance that the switch could have a slight transition delay to make it more natural like the one on [this site](https://philipptanlak.com/) (the switch button is in the footer's top-right section

I can make a PR for this

**Comment by kdkasad** [comment]
I like the idea. Go ahead and make a PR so we can see the result and any possible side-effects.

**Reply by orimdominic** [reply]
Awesome! Sending one soonest!

**Reply by natalie-o-perret** [reply]
@orimdominic 👀

**Comment by orimdominic** [comment]
@natalie-o-perret you may be interested in this https://github.com/adityatelange/hugo-PaperMod/pull/407
